CHANGELOG.md

```diff
@@ -34,6 +34,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * The `git.push-branch-prefix` config has been deprecated in favor of
   `git.push-bookmark-prefix`.
 
+* `conflict()` and `file()` revsets have been renamed to `conflicts()` and `files()`
+  respectively. With this, all revsets that could potentially return multiple
+  results are named in the plural.
```

Comment:
I'm not sure I understood @yuja's point correctly, but I think he was also saying that the last sentence isn't quite correct. We have e.g. `description()` and `author()` that are not named in the plural.

Reply:
I've just removed that sentence. The deprecation notice is enough by itself. The issue discussions have more detail than I was trying to cram into one ambigous and wrong sentence. thanks!